ui/page/privacy/manual_mixer_setup_page.go

```diff
@@ -122,65 +119,123 @@ func (pg *ManualMixerSetupPage) OnNavigatedTo() {
+										layout.Rigid(func(gtx C) D {
+											return layout.Spacer{Height: values.MarginPadding10}.Layout(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											return pg.cautionCard(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											return layout.Spacer{Height: values.MarginPadding40}.Layout(gtx)
+										}),
+									)
 								}),
 								layout.Rigid(func(gtx C) D {
-									return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding16, Right: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
-										return layout.Flex{
-											Axis: layout.Horizontal,
-										}.Layout(gtx,
-											layout.Rigid(func(gtx C) D {
-												return pg.Theme.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
-											}),
-											layout.Rigid(func(gtx C) D {
-												txt := `<span style="text-color: grayText2">
-											<b>Make sure to select the same accounts from the previous privacy setup. </b><br>Failing to do so could compromise wallet privacy.<br> You may not select the same account for mixed and unmixed.
-										</span>`
-												return layout.Inset{
-													Left: values.MarginPadding8,
-												}.Layout(gtx, renderers.RenderHTML(txt, pg.Theme).Layout)
-											}),
-										)
-									})
+									gtx.Constraints.Min.X = gtx.Constraints.Max.X
+									return layout.UniformInset(values.MarginPadding15).Layout(gtx, pg.toPrivacySetup.Layout)
 								}),
 							)
-						}),
-						layout.Rigid(func(gtx C) D {
-							gtx.Constraints.Min.X = gtx.Constraints.Max.X
-							return layout.UniformInset(values.MarginPadding15).Layout(gtx, pg.toPrivacySetup.Layout)
-						}),
-					)
-				})
-			},
-		}
-		return page.Layout(pg.ParentWindow(), gtx)
-	}
+						})
+					})
+				}),
+			)
+		})
+	})
+}
 
-	return components.UniformPadding(gtx, body)
+func (pg *ManualMixerSetupPage) cautionCard(gtx C) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.Inset{Left: values.MarginPadding15, Right: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+		card := pg.Theme.Card()
+		card.Color = pg.Theme.Color.Gray4
+		return card.Layout(gtx, func(gtx C) D {
+			gtx.Constraints.Min.X = gtx.Constraints.Max.X
+			gtx.Constraints.Min.Y = int(values.MarginPadding100)
+			gtx.Constraints.Max.Y = gtx.Constraints.Min.Y
+			return layout.Inset{Top: values.MarginPadding15, Left: values.MarginPadding15,
+				Right: values.MarginPadding15, Bottom: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+				return layout.Flex{Alignment: layout.Start}.Layout(gtx,
+					layout.Flexed(1, func(gtx C) D {
+						return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle, Spacing: layout.SpaceAround}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								gtx.Constraints.Max.X = int(values.MarginPadding40)
+								return pg.Theme.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
+							}),
+						)
+					}),
+					layout.Flexed(7, func(gtx C) D {
```

Comment:
Flexed(1) and Flexed(7) seems off.

Reply:
What would be good values here.

Reply:
@JoeGruffins  I think it just:

```
return layout.Flex{Alignment: layout.Start}.Layout(gtx,
					layout.Rigid(func(gtx C) D {
						gtx.Constraints.Max.X = int(values.MarginPadding40)
						return layout.Center.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
							return pg.Theme.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
						})
					}),
					layout.Rigid(func(gtx C) D {
						return layout.Inset{
							Left: values.MarginPadding10,
						}.Layout(gtx, func(gtx C) D {
							return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
								layout.Rigid(func(gtx C) D {
									label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleWarningTitle))
									return label.Layout(gtx)
								}),
								layout.Rigid(func(gtx C) D {
									label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleWarningDesc))
									return label.Layout(gtx)
								}),
							)
						})
					}),
				)
```

this is my suggestion code.

Reply:
Ok.